### DISCUSSION TITLE
Play videos from Linux-Show-Player - Success!

### DISCUSSION THREAD

**Discussion by MarleyThompson** [FrancescoCeruti/linux-show-player#323] [General]
Today, I had the requirement to play a video from Show Player. Of course, you can create a media cue with the video filename. But that just plays the sound in the video. It does not play the video itself.
No problem, create a command cue and enter a command to play the video. I used MPV. Here is a typical command:

`mpv --screen=1 --fs --fs-screen=1 ~/Music/SchoolPlay/Performance/OpeningLink1.mov`

This opens the video full-screen in the secondary screen (the projector). I have the background of this screen set to black so that the projector is dark when not playing.
And yes! You can play another audio cue while the video is playing.
Perhaps you already knew this but I count it as a success all round.
Many thanks for this application - appreciated!
Linux Mint 21.3 Cinnamon

**Comment by fnetX** [comment]
You might also want to add `--no-osc --no-osd-bar --osd-font-size=1` to prevent showing menues when inadvertently hovering over the video player :)
